Makefile

```diff
@@ -100,6 +100,7 @@ define _SOURCES:=
 	mgmt.c
 	module.c
 	module_init.c
+	command_hooker.c
```

Comment:
https://www.google.com/search?q=hooker
:-)

`command_hook.c` is better.

Reply:
Fixed